roles/configure_ztp_gitops_apps/tasks/main.yaml

```diff
@@ -103,30 +124,40 @@
         - "  - group: hive.openshift.io"
 
     - name: Add ClusterImageSet to the policies-app-project.yaml
-      lineinfile:
+      ansible.builtin.lineinfile:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/policies-app-project.yaml"
         line: "{{ item }}"
         insertafter: clusterResourceWhitelist.*
       loop:
         - "    kind: ClusterImageSet"
         - "  - group: hive.openshift.io"
 
-    - name: Run the policies and cluster apps
-      shell: |
-        {{ czga_oc_tool_path }} --kubeconfig="{{ temp_dir.path }}/kubeconfig" apply -k "{{ temp_dir.path }}/ztp/argocd/deployment"
-
     - name: Replace image container URL in argocd-openshift-gitops-patch.json
-      replace:
+      ansible.builtin.replace:
         path: "{{ temp_dir.path }}/ztp/argocd/deployment/argocd-openshift-gitops-patch.json"
         regexp: "{{ ztp_site_generator_image }}.*/"
         replace: "{{ ztp_site_generator_image }}:{{ ztp_site_generator_version }}"
 
     - name: Apply ArgoCD changes
-      shell: |
+      ansible.builtin.shell: |
```

Comment:
The name of this task should be a bit different, something like "Patch the ArgoCD instance to install the GitOps ZTP plugin", just to follow what the docs says: https://docs.openshift.com/container-platform/4.15/edge_computing/ztp-preparing-the-hub-cluster.html#ztp-configuring-hub-cluster-with-argocd_ztp-preparing-the-hub-cluster

Reply:
Done.